src/node/utils/Minify.js

```diff
@@ -141,19 +141,11 @@ const minify = async (req, res) => {
 
   // Backward compatibility for plugins that were written when jQuery lived at
   // src/static/js/jquery.js.
-  if (['js/jquery.js', 'plugins/ep_etherpad-lite/static/js/jquery.js'].indexOf(filename) !== -1) {
+  if (filename.endsWith('js/jquery.js')) {
```

Comment:
This matches `foojs/jquery.js`, which I don't think is a good idea. I think it's better to leave this line as-is.

Reply:
OK good point